playbooks/robusta_playbooks/krr.py

```diff
@@ -209,6 +251,8 @@ def krr_scan(event: ExecutionBaseEvent, params: KRRParams):
                             "request": scan.recommended.requests[resource].priority,
                             "limit": scan.recommended.limits[resource].priority,
                         },
+                        "metric": scan.metrics[resource].dict(),
```

Comment:
There will always be metrics for every resource? (no risk of null pointer error)

Reply:
also this is from rebase, not added by me